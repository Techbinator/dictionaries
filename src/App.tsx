import React from "react";
import { BrowserRouter, Route } from "react-router-dom";

import Header from "./components/Header";
import Dictionaries from "./pages/Dictionaries";
import Dataset from "./pages/Dataset";
import { DictionariesProvider } from "./store/DictionariesContext";

import "./App.scss";

class App extends React.Component {
  render() {
    return (
      <BrowserRouter>
        <DictionariesProvider>
          <Header />
          <div className="content">
            <Route exact path="/" component={Dataset} />
            <Route exact path="/dictionaries" component={Dictionaries} />
          </div>
        </DictionariesProvider>
      </BrowserRouter>
    );
  }
}

export default App;
