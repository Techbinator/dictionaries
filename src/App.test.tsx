import React from "react";
import App from "./App";
import { shallow } from "enzyme";

describe("App component", () => {
  it("should render snapshot with default values", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
