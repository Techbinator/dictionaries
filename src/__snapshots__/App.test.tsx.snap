// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render snapshot with default values 1`] = `
<BrowserRouter>
  <DictionariesProvider>
    <_default />
    <div
      className="content"
    >
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/dictionaries"
      />
    </div>
  </DictionariesProvider>
</BrowserRouter>
`;
